src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectPropertyDescriptor.cs

```diff
@@ -221,7 +246,7 @@ private EventDescriptor ChangedEventValue
             {
                 if (!_state[s_bitChangedQueried])
                 {
-                    _realChangedEvent = TypeDescriptor.GetEvents(ComponentType)[Name + "Changed"];
+                    _realChangedEvent = TypeDescriptor.GetEvents(_componentClass)[Name + "Changed"];
```

Comment:
Shouldn't we also annotate the PropertyGetter?

Reply:
We can't because it is an override. If you apply it to a virtual property, it needs to be applied to the whole hierarchy.